orangecontrib/prototypes/widgets/owfeaturestatistics.py

```diff
@@ -0,0 +1,512 @@
+import locale
+from enum import auto, IntEnum
+from functools import partial
+from itertools import chain
+
+import numpy as np
+import scipy.stats as ss
+from AnyQt.QtCore import Qt, QSize, QRectF, QAbstractTableModel, QVariant, \
+    QModelIndex, pyqtSlot, QSortFilterProxyModel, QRegExp
+from AnyQt.QtGui import QPainter, QColor, QKeySequence
+from AnyQt.QtWidgets import QStyleOptionViewItem, QShortcut
+from AnyQt.QtWidgets import QStyledItemDelegate, QGraphicsScene, \
+    QTableView, QHeaderView, QStyle
+# pylint: disable=unused-import
+from typing import Any, Optional
+
+import Orange.statistics.util as ut
+from Orange.canvas.report import plural
+from Orange.data import Table, StringVariable, DiscreteVariable, \
+    ContinuousVariable, TimeVariable
+from Orange.widgets import widget, gui
+from Orange.widgets.settings import Setting, ContextSetting
+from Orange.widgets.utils.itemmodels import DomainModel
+from Orange.widgets.utils.signals import Input
+from orangecontrib.prototypes.widgets.utils.histogram import Histogram
+
+
+class FeatureStatisticsTableModel(QAbstractTableModel):
```

Comment:
You may be able to extend `itemmodels.AbstractSortTableModel` for fast numpy sorting, but not (yet) for filtering.